### PR TITLE
added watchers to std lib

### PR DESCRIPTION
**Pull request by ibdthor**

### CODE REVIEW

File: v0.3/handbook/libraries/stdlib.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
 ---
-layout: default
```
Comment (cmontella) [inline v0.3/handbook/libraries/stdlib.md]: I don't think you can remove this.... not entirely sure though. It tells Jekyll which layout to use.

**Review by cmontella — DISMISSED** [review]

File: v0.3/handbook/libraries/stdlib.md (hunk 1 above)
Comment (joshuafcole) [reply v0.3/handbook/libraries/stdlib.md]: :+1:

File: v0.3/handbook/libraries/stdlib.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
 
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+
 # Standard Library
 
-## Aggregates
+##  Aggregates
```
Comment (joshuafcole) [inline v0.3/handbook/libraries/stdlib.md]: Remove this extra space.

File: v0.3/handbook/libraries/stdlib.md [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
 
 <table>
-
   <tr>
-    <td colspan="2">
```
Comment (joshuafcole) [inline v0.3/handbook/libraries/stdlib.md]: These need to be colspan 2 or they look funny. For example: https://github.com/cmontella/docs/blob/e788fcd80d5a2b9873677799b0030d0481707387/v0.3/handbook/libraries/stdlib.md

File: v0.3/handbook/libraries/stdlib.md [hunk 4]
```diff
@@ -387,118 +391,979 @@ x = random/number[seed: seconds]</code>
...
+        <li><strong>tagname</strong> - determines the type of html element rendered in the browser; can be any html element given as a string</li>
+        <li><strong>style</strong> - (optional) - specific CSS styles can be defined, but must be entered as a subrecord</li>
+        <li><strong>text</strong> - (optional) - defines text within the html element</li>
+        <li><strong>#</strong> - (optional) - any other tags on the record will be applied to the element as classes</li>
+        <li><strong>class</strong> - (optional) defines one or more classes for the element</li>
+        <li>Other attributes - (optional) - other attribute-value pairs will be applied directly to the element for integration with existing JS libraries or debugging</li>
+      </ul>
+    </td>
+    <td>
+      <code>// commits a div in the browser with the text “Hello world!” and the class “hello”
```
Comment (joshuafcole) [inline v0.3/handbook/libraries/stdlib.md:506]: These code blocks aren't wrapping and don't have whitespace included, so they look pretty bad. https://github.com/cmontella/docs/blob/e788fcd80d5a2b9873677799b0030d0481707387/v0.3/handbook/libraries/stdlib.md#html

File: v0.3/handbook/libraries/stdlib.md [hunk 5]
```diff
@@ -387,118 +391,979 @@ x = random/number[seed: seconds]</code>
...
+```
+commit
+  [#html/element #hello tagname: “div” text: “Hello world!”]
+```</code>
```
Comment (joshuafcole) [inline v0.3/handbook/libraries/stdlib.md:510]: don't need both \<code\> and \`\`\`. That's a recipe for sadness. Look at how messed up the document layout is here.

**Review by joshuafcole — DISMISSED** [review]
Busted formatting makes it very hard to review past halfway through HTML. Will do another review once that's fixed.

**Pull request merged**